tests/snuba/api/endpoints/test_discover_key_transactions.py

```diff
@@ -0,0 +1,190 @@
+        assert key_transaction.organization == self.org
+
+    def test_save_with_wrong_project(self):
+        other_user = self.create_user()
+        other_org = self.create_organization(owner=other_user)
+        other_project = self.create_project(organization=other_org)
+
+        data = load_data("transaction")
+        with self.feature("organizations:discover"):
+            url = reverse("sentry-api-0-organization-key-transactions", args=[self.org.slug])
+            response = self.client.post(
+                url, {"transaction": data["transaction"], "project": other_project.id}
+            )
+
+        assert response.status_code == 400
+        assert response.data["detail"] == "No project with that id found"
+
+    def test_max_key_transaction(self):
+        data = load_data("transaction")
+        for i in range(MAX_KEY_TRANSACTIONS):
+            KeyTransaction.objects.create(
+                owner=self.user,
+                organization=self.org,
+                transaction=data["transaction"] + six.text_type(i),
+                project=self.project,
+            )
+        with self.feature("organizations:discover"):
+            url = reverse("sentry-api-0-organization-key-transactions", args=[self.org.slug])
+            response = self.client.post(
+                url, {"transaction": data["transaction"], "project": self.project.id}
+            )
+
+        assert response.status_code == 400
+        assert response.data["detail"] == "At most {} Key Transactions can be added".format(
+            MAX_KEY_TRANSACTIONS
+        )
+
+    def test_get_key_transactions(self):
+        project2 = self.create_project(name="foo", organization=self.org)
+        event_data = load_data("transaction")
+
+        transactions = [
+            (self.project, "/foo_transaction/"),
+            (self.project, "/blah_transaction/"),
+            (self.project, "/zoo_transaction/"),
+            (project2, "/bar_transaction/"),
+        ]
+
+        for project, transaction in transactions:
+            event_data["transaction"] = transaction
+            self.store_event(data=event_data, project_id=project.id)
+            KeyTransaction.objects.create(
+                owner=self.user,
+                organization=self.org,
+                transaction=event_data["transaction"],
+                project=project,
+            )
+
+        with self.feature("organizations:discover"):
+            url = reverse("sentry-api-0-organization-key-transactions", args=[self.org.slug])
+            response = self.client.get(
+                url,
+                {
+                    "project": [self.project.id, project2.id],
+                    "orderby": "transaction",
+                    "field": [
+                        "transaction",
+                        "transaction_status",
+                        "project",
+                        "rpm()",
+                        "error_rate()",
+                        "p95()",
```

Comment:
percentile(transaction.duratio
```suggestion
                        "percentile(transaction.duration, 0.95)",
```